--- libav-10/configure.orig	2014-03-29 19:25:18.992079006 +0100
+++ libav-10/configure	2014-03-29 19:47:21.502023509 +0100
@@ -2040,7 +2040,7 @@
 avplay_libs='$sdl_libs'
 avplay_select="rdft"
 avprobe_deps="avcodec avformat"
-avserver_deps="avformat fork !shared"
+avserver_deps="avformat fork static"
 avserver_select="ffm_muxer rtp_protocol rtsp_demuxer"
 
 # documentation
--- libav-9.1/Makefile.orig	2013-01-07 11:17:21.000000000 +0100
+++ libav-9.1/Makefile	2013-01-12 10:58:43.537450899 +0100
@@ -137,6 +137,8 @@
 
 $(foreach P,$(PROGS-yes),$(eval $(call DOPROG,$(P))))
 
+avserver: FF_EXTRALIBS=-lavdevice -lavfilter libavformat/libavformat.a -lavresample -lavcodec -lswscale -lavutil $(EXTRALIBS)
+
 $(PROGS): %$(EXESUF): %.o $(FF_DEP_LIBS)
 	$(LD) $(LDFLAGS) $(LD_O) $(OBJS-$*) $(FF_EXTRALIBS)
 
