--- ffmpeg-1.0.1/libavdevice/libcdio.c.orig	2012-12-03 22:17:35.000000000 +0100
+++ ffmpeg-1.0.1/libavdevice/libcdio.c	2012-12-06 19:03:32.695282861 +0100
@@ -23,8 +23,8 @@
  * libcdio CD grabbing
  */
 
-#include <cdio/cdda.h>
-#include <cdio/paranoia.h>
+#include <cdio/paranoia/cdda.h>
+#include <cdio/paranoia/paranoia.h>
 
 #include "libavutil/log.h"
 #include "libavutil/mem.h"
--- libav-9.1/configure.orig	2013-01-10 21:00:22.840300169 +0100
+++ libav-9.1/configure	2013-01-10 21:29:21.180263938 +0100
@@ -3552,7 +3552,7 @@
 enabled_any sndio_indev sndio_outdev && check_lib2 sndio.h sio_open -lsndio
 
 enabled libcdio &&
-    check_lib2 "cdio/cdda.h cdio/paranoia.h" cdio_cddap_open -lcdio_paranoia -lcdio_cdda -lcdio
+    check_lib2 "cdio/paranoia/cdda.h cdio/paranoia/paranoia.h" cdio_cddap_open -lcdio_paranoia -lcdio_cdda -lcdio
 
 enabled x11grab                                           &&
 require X11 X11/Xlib.h XOpenDisplay -lX11                 &&
