--- libav-9.1/configure.orig	2013-01-10 21:32:17.613593592 +0100
+++ libav-9.1/configure	2013-01-11 22:20:57.565065216 +0100
@@ -1782,7 +1782,7 @@
 avplay_deps="avcodec avformat avresample swscale sdl"
 avplay_select="rdft"
 avprobe_deps="avcodec avformat"
-avserver_deps="avformat ffm_muxer fork rtp_protocol rtsp_demuxer !shared"
+avserver_deps="avformat ffm_muxer fork rtp_protocol rtsp_demuxer static"
 avserver_extralibs='$ldl'
 
 doc_deps="texi2html"
--- libav-9.1/Makefile.orig	2013-01-07 11:17:21.000000000 +0100
+++ libav-9.1/Makefile	2013-01-12 10:58:43.537450899 +0100
@@ -137,6 +137,8 @@
 
 $(foreach P,$(PROGS-yes),$(eval $(call DOPROG,$(P))))
 
+avserver: FF_EXTRALIBS=-lavdevice -lavfilter libavformat/libavformat.a -lavresample -lavcodec -lswscale -lavutil $(EXTRALIBS)
+
 $(PROGS): %$(EXESUF): %.o $(FF_DEP_LIBS)
 	$(LD) $(LDFLAGS) $(LD_O) $(OBJS-$*) $(FF_EXTRALIBS)
 
