--- libav-9.1/configure.orig	2013-01-10 20:51:10.110311689 +0100
+++ libav-9.1/configure	2013-01-10 20:55:14.063639939 +0100
@@ -3467,7 +3467,7 @@
 enabled libfaac    && require2 libfaac "stdint.h faac.h" faacEncGetVersion -lfaac
 enabled libfdk_aac && require  libfdk_aac fdk-aac/aacenc_lib.h aacEncOpen -lfdk-aac
 enabled libfreetype && require_pkg_config freetype2 "ft2build.h freetype/freetype.h" FT_Init_FreeType
-enabled libgsm     && require  libgsm gsm/gsm.h gsm_create -lgsm
+enabled libgsm     && require  libgsm gsm.h gsm_create -lgsm
 enabled libilbc    && require  libilbc ilbc.h WebRtcIlbcfix_InitDecode -lilbc
 enabled libmp3lame && require  "libmp3lame >= 3.98.3" lame/lame.h lame_set_VBR_quality -lmp3lame
 enabled libopencore_amrnb  && require libopencore_amrnb opencore-amrnb/interf_dec.h Decoder_Interface_init -lopencore-amrnb
--- ffmpeg-0.10/libavcodec/libgsm.c.orig	2012-01-26 23:15:59.000000000 +0100
+++ ffmpeg-0.10/libavcodec/libgsm.c	2012-01-27 17:58:28.495275196 +0100
@@ -27,7 +27,7 @@
 
 // The idiosyncrasies of GSM-in-WAV are explained at http://kbs.cs.tu-berlin.de/~jutta/toast.html
 
-#include <gsm/gsm.h>
+#include <gsm.h>
 
 #include "avcodec.h"
 #include "gsm.h"
