--- libav-9.1/configure.orig	2013-01-10 20:55:39.673639404 +0100
+++ libav-9.1/configure	2013-01-10 20:59:51.676967488 +0100
@@ -3473,7 +3473,7 @@
 enabled libopencore_amrnb  && require libopencore_amrnb opencore-amrnb/interf_dec.h Decoder_Interface_init -lopencore-amrnb
 enabled libopencore_amrwb  && require libopencore_amrwb opencore-amrwb/dec_if.h D_IF_init -lopencore-amrwb
 enabled libopencv  && add_extralibs -lopencv_core -lopencv_features2d -lopencv_imgproc -lopencv_objdetect
-enabled libopenjpeg && require libopenjpeg openjpeg.h opj_version -lopenjpeg
+enabled libopenjpeg && require_pkg_config libopenjpeg openjpeg.h opj_version
 enabled libopus    && require_pkg_config opus opus_multistream.h opus_multistream_decoder_create
 enabled libpulse && require_pkg_config libpulse-simple pulse/simple.h pa_simple_new
 enabled librtmp    && require_pkg_config librtmp librtmp/rtmp.h RTMP_Socket
