--- libav-11.4/libavcodec/libilbc.c.orig	2015-05-31 16:56:19.000000000 +0200
+++ libav-11.4/libavcodec/libilbc.c	2015-10-17 10:49:08.164415994 +0200
@@ -95,8 +95,8 @@
         return ret;
     }
 
-    WebRtcIlbcfix_DecodeImpl((WebRtc_Word16*) frame->data[0],
-                             (const WebRtc_UWord16*) buf, &s->decoder, 1);
+    WebRtcIlbcfix_DecodeImpl((int16_t*) frame->data[0],
+                             (const uint16_t*) buf, &s->decoder, 1);
 
     *got_frame_ptr = 1;
 
@@ -168,7 +168,7 @@
         return ret;
     }
 
-    WebRtcIlbcfix_EncodeImpl((WebRtc_UWord16*) avpkt->data, (const WebRtc_Word16*) frame->data[0], &s->encoder);
+    WebRtcIlbcfix_EncodeImpl((uint16_t*) avpkt->data, (const int16_t*) frame->data[0], &s->encoder);
 
     avpkt->size     = s->encoder.no_of_bytes;
     *got_packet_ptr = 1;
